vector/src/main/res/values/strings.xml

```diff
@@ -3416,11 +3416,15 @@
     <string name="space_explore_activity_title">Explore rooms</string>
     <string name="space_add_child_title">Add rooms</string>
     <string name="leave_space">Leave Space</string>
-    <string name="space_leave_prompt_msg">Are you sure you want to leave the space?</string>
+    <string name="space_leave_prompt_msg">Are you sure you want to leave %s?</string>
```

Comment:
The existing translations if any will fail :/

Reply:
deleted the old one, and added a new one